blog/_posts/2021-07-01-June-2021-update.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "June 2021 Update"
+description: ""
+tags: []
+---
+{% include JB/setup %}
+
+Dear GeoCAT Community,
+
+Our team would like to share June, 2021 update 
+about GeoCAT releases and community activities below:  
+
+### - GeoCAT-comp v2021.06.0 Released!
+With this release, optimizations to the `month_to_season` function and 
+bug-fixing to `interp_hybrid_to_sigma` function have been made available.
+
+- Please see
+  [GeoCAT-comp v2021.06.0 Release](https://github.com/NCAR/geocat-comp/releases/tag/v2021.06.0)
+  for further information about Release and Developer Notes.
+- Refer to
+  [GeoCAT-comp API reference](https://geocat-comp.readthedocs.io/en/latest/api.html)
+  for detailed information about the GeoCAT-comp functions.
+
+### - GeoCAT-f2py v2021.06.0 Released!
+With this release, slight build environment maintenance has been performed.
+
+- Please see 
+[GeoCAT-f2py v2021.06.0 Release](https://github.com/NCAR/geocat-f2py/releases/tag/v2021.06.0)
+for further information about Release and Developer Notes.
+
+- Please see our 
+[Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) 
+for detailed information about our development stack. 
+
+## Community Events

```

Comment:
```suggestion
### - Community Events

```

Reply:
Thanks for this suggestion; it led me rethink about the sizes and fonts of all the titles, and I have applied a revision to all of them for May and June updates. If we like them, I will do the same change for previous monthly updates as well.